src/actions/API.actions.js

```diff
@@ -95,6 +96,8 @@ export function createSUSIMessage(createdMessage, currentThreadID, voice) {
       Actions.sendToHardwareDevice(response);
 
       receivedMessage.text = response.answers[0].actions[0].expression;
+      // Setting Language received from User
+      receivedMessage.lang = response.answers[0].actions[0].language;
```

Comment:
Add a check to see if response.answers[0].actions[0] contains language attribute or not. Rest looks good 👍